src/app/app.module.ts

```diff
@@ -57,9 +60,10 @@ import { AppRoutingModule } from './app-routing.module';
     AccountingModule,
     SelfServiceModule,
     SystemModule,
+    OrganizationModule,
     AppRoutingModule
   ],
-  declarations: [WebAppComponent],
+  declarations: [WebAppComponent, OrganizationComponent],
```

Comment:
Why have you declared this in the main app module? What is the use of organization module?